Content search query fixed, zero state logic added, documentation upd… (#151)
rapid7/insightconnect-workflows: workflows/Purge_Office_365_Emails_with_Slack/help.md

Sample Slack Trigger Commands:

> `@Rapid7 InsightConnect purge-email subject="A phishy email"`

Reviewer: ```suggestion
`@Slackbot purge-email subject="A phishy email"`
```

> `@Rapid7 InsightConnect purge-email subject="A phishy email" from="example.com" body="Click here for free stuff"`

Reviewer: ```suggestion
`@Slackbot purge-email subject="A phishy email" from="example.com" body="Click here for free stuff"`
```

> `@Rapid7 InsightConnect purge-email subject="A phishy email" delete=true`

Reviewer: ```suggestion
`@Slackbot purge-email subject="A phishy email" delete=true`
```

# Version History

> * 1.0.2 - Changed content search query to use double quotes. Workflow no longer prompts a manual purge when 0 emails are found. Trigger no longer requires an exclamation mark. Updated documatation

Reviewer 2: ```suggestion
* 1.0.2 - Changed content search query to use double quotes | Workflow no longer prompts a manual purge when 0 emails are found | Trigger no longer requires an exclamation mark | Updated documentation
```
